test/app-tap/tarantoolctl.test.lua

```diff
@@ -111,10 +112,11 @@ local function tctl_wait_stop(dir, name)
 end
 
 local function tctl_command(dir, cmd, args)
+    local rocks_path = SOURCEDIR .. '/third_party/luarocks/src/?.lua;'
```

Comment:
AFAIU, we generally should end `LUA_PATH` with `;;` (double semicolon) to don't discard default search paths.

Reply:
fixed